### ISSUE TITLE
Backdrop's GestureDetector should be excluded from semantics

### ISSUE THREAD

**Issue by tony-ditchlabs** [akshathjain/sliding_up_panel#341]
**Describe the bug**
Using the package `accessibility_tools`, the package flags a backdrop where `backdropTapClosesPanel` is true as missing a semantics label for screen readers / other assistive tech. A minor headache, but automated tests that depend on it are failing.

**Workaround**
Setting `backdropTapClosesPanel` to false removes the `GestureDetector` from the widget tree, but it's not great UX.

**To Fix**
Add `excludeFromSemantics: true` to the backdrop's `GestureDetector`.

**Comment by tony-ditchlabs** [comment]
Closing, duplicate of [168](https://github.com/akshathjain/sliding_up_panel/issues/168).
